app/build.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.appcompat:appcompat:1.0.2'
-    implementation 'androidx.core:core-ktx:1.0.2'
+    implementation 'androidx.core:core-ktx:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sadashi-ota/android-bitrise-sample/blob/91337411dde06a0fbb02ad02dc05ade41d3a026f/app/build.gradle#L39"></span>A newer version of androidx.constraintlayout:constraintlayout than 1.1.3 is available: 2.1.1</td>
    </tr>
  </tbody>
</table>